style.css

```diff
@@ -1,136 +1,94 @@
-ul {
-    list-style: none;
-    margin: 0;
-    padding-left: 0;
-    justify-content: right;
-    padding-right: 8px;
-    padding-right: 50px;
-    
+.logo img {
+    width: 107.54px;
+    height: 16.71px;
 }
 
-a {
-    text-decoration: none;
-    color: #4D4D4D;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-    vertical-align: middle;
-    font-size:small;
-    line-height: 1%;
-
-
+.header {
+    background-color: white;
+    justify-content: space-between;
+    display: flex;
 }
 
-li {
-    display: inline;
-    padding-bottom: 200px;
-    
+body {
+    background-color: black;
 }
 
+ul li {
+    display: inline-block;
+}
 
-
-header {
-    display: flex;
-    width: 1002.34;
-    height: 100%;
-    padding-left: 75px;
-    padding-right: 75px;
-    justify-content: space-between;
- 
-
-
-    
+ul {
+    display: inline-block;
 }
 
-nav {
+li a {
     text-decoration: none;
-    list-style: none;
-    display: flex;
-    justify-content: space-between;
-    line-height: 30px;
-    
-   
-    
-
-
+    color: #4D4D4D;
+    padding: 16.71px;
 }
-.buttom{
-    background-color: #4CAF50;
-    border-radius: 3px;
+
+.button {
+    background-color: #4CAF4F;
     color: white;
-    padding: 12px 20px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 11.14;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    border-radius: 2.78px;
 }
 
-nav a{
-    padding: 5px;
-    line-height: 13px;
-
-
-   
-}
-.buttom a{
-    vertical-align: 20px;
+.RightArrow {
+    vertical-align: baseline;
+    height: 11.14px;
+    width: 11.14px;
 }
-.les{
+
+a,
+h1,
+h2,
+p {
+    font-family: 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
     color: #4D4D4D;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
 }
-#from{
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+
+h2 span {
     color: #4CAF4F;
+    font-size: 44.55px;
 }
-.greytext{
-    color: #4D4D4D;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-    font-size: 11px;
-}
-#reg{
-    background-color: #4CAF50;
-    border-radius: 3px;
-    color: white;
-    padding: 17px 22px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 11.14;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+
+.block1 h2 {
+    font-size: 44.55px;
 }
```

Comment:
Use proper order of styles. Like:
1. .block1
2. .block1 h2 (because h2 is inside this block)
3. .block1 h2 span (because span is inside h2 that is inside block)

`.css` files should copy structure of HTML file